### DISCUSSION TITLE
How do you change compile defaults and add custom compile options

### DISCUSSION THREAD

**Discussion by thomprl** [codefori/vscode-ibmi#1865] [Q&A]
How would I go about changing the compile defaults for CRTSQLRPGI for example and is there a way to add custom actions?

**Comment by worksofliam** [comment]
Hey @thomprl !

Check out this page: https://codefori.github.io/docs/#/pages/developing/actions/index

**Reply by thomprl** [reply]
Thank you.. I could see this being a global type setting like you were talking about.  We manually set our TARGET release to the version of OS we are currently supporting.  Hate to have to do this 20 times for everyone.  But not sure that's possible or not.

**Reply by worksofliam** [reply]
CC @chrjorgensen another call for server side settings.

**Reply by m-tyler** [reply]
I am going to add on to @thomprl's ask.  We use a special library to override the target release and I had to modify my actions to prevent compile problems.

-Matt
